db/schema.rb

```diff
@@ -47,7 +44,7 @@
     t.index ["url"], name: "index_subjects_on_url"
   end
 
-  create_table "users", id: :serial, force: :cascade do |t|
+  create_table "users", id: :integer, force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4" do |t|
```

Comment:
The schema seems to change whenever I switch back and forth :( I'll try to change it so that this doesnt happen